templates/blocks/author_profile.twig

```diff
@@ -6,9 +6,25 @@
 				<h5 rel="author" class="author-block-info-name">
 					<a href="{{ post.author.link }}" itemprop="name">{{ post.author.name }}</a>
 				</h5>
+
+				{% set author_bio_readmore %}
+					...  &nbsp;<a href="{{ post.author.link }}">{{ __( 'Read More', 'planet4-master-theme' ) }}</a>
+				{% endset %}
+
+				{% if ( fn('wp_is_mobile') ) %}
+					{% set author_bio_char_limit = 180 %}
+					{% set author_bio = fn('substr', post.author.description, 0, fn('strpos', post.author.description ,' ', author_bio_char_limit)) %}
+					{% set author_bio = post.author.description|length > author_bio_char_limit ? author_bio ~ author_bio_readmore : author_bio %}
+				{% else %}
+					{% set author_bio_char_limit = 300 %}
+					{% set author_bio = fn('substr', post.author.description, 0, fn('strpos', post.author.description ,' ', author_bio_char_limit)) %}
+					{% set author_bio = post.author.description|length > author_bio_char_limit ? author_bio ~ author_bio_readmore : author_bio %}
```

Comment:
You can take the last 2 lines out of the if-statement as they are the same in both cases